uint8 currentCVTask

uint8 NOTHING=0
uint8 GATE=1

#define constants for other tasks here